paddle/fluid/framework/ir/mkldnn/cpu_quantize_placement_pass_tester.cc

```diff
@@ -96,38 +94,36 @@ void MainTest(std::initializer_list<std::string> quantize_enabled_op_types,
 
   graph.reset(pass->Apply(graph.release()));
 
-  unsigned use_quantizer_true_count = 0;
+  unsigned int8_data_type_count = 0;
 
   for (auto* node : graph->Nodes()) {
     if (node->IsOp()) {
       auto* op = node->Op();
-      if (op->HasAttr("use_quantizer") &&
-          BOOST_GET_CONST(bool, op->GetAttr("use_quantizer"))) {
-        ++use_quantizer_true_count;
+      if (op->GetAttrIfExists<std::string>("mkldnn_data_type") == "int8") {
```

Comment:
Can `HasOpINT8DataType()` be used here?

Reply:
I think it would improve readability. Moreover, later we will probably need something like `HasOpBF16DataType()` and having both in same place (`mkldnn_helper.h`) is a good idea.